sphinx_scality/static/js/searchtools.js

```diff
@@ -26,7 +26,7 @@ if (!Scorer) {
     // query matches the full name of an object
     objNameMatch: 11,
     // or matches in the last dotted part of the object name
-    objPartialMatch: 6,
+    objPartialMatch: 15,
```

Comment:
I don't understand this change. What does it do? Would be worth explaining in the commit message.